tests/fixture/tmpnet/flags/kube_config.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package flags
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/spf13/pflag"
+
+	"github.com/ava-labs/avalanchego/tests/fixture/tmpnet"
+)
+
+const KubeConfigPathEnvVar = "KUBECONFIG"
+
+type KubeconfigVars struct {
+	Path    string
+	Context string
+}
+
+// NewKubeconfigFlagVars registers kubeconfig flag variables for stdlib flag
+func NewKubeconfigFlagVars() *KubeconfigVars {
+	return newKubeconfigFlagVars("")
+}
+
+// internal method enabling configuration of the doc prefix
+func newKubeconfigFlagVars(docPrefix string) *KubeconfigVars {
+	v := &KubeconfigVars{}
+	v.register(flag.StringVar, docPrefix)
+	return v
+}
```

Comment:
Is there a point having this unexported function with docPrefix argument, if the prefix is always the empty string? Or is it to be used in an upcoming PR? Thanks!